scripts/parse-generated-api.ts

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const path = require('path');
+
+// Read the content of the file
+const filePath = path.resolve(__dirname, '../public/app/features/provisioning/api/endpoints.gen.ts');
+const outputFilePath = path.resolve(__dirname, '../public/app/features/provisioning/api/endpoints.ts');
+const content = fs.readFileSync(filePath, 'utf-8');
+
+// Define the prefixes to remove
+const prefixes = [
+  'ComGithubGrafanaGrafanaPkgApisProvisioningV0Alpha1',
```

Comment:
Is this file provisioning specific?  Or should we take a list of possible inputs?